### DISCUSSION TITLE
Sysmon and Suricata

### DISCUSSION THREAD

**Discussion by WPTechnician** [Security-Onion-Solutions/securityonion#9007] [General] [answered]
Hi, just had a quick question.  If I push sysmon to my endpoints and have them sending those to SO, will they raise Suricata alerts?

Thanks!

**Comment by dougburks** [comment, marked as answer]
No, Sysmon and Suricata are totally independent of each other. Sysmon generates telemetry about what is happening on an endpoint, whereas Suricata analyzes network traffic from a tap or span port. For more information, please see:
https://docs.securityonion.net/en/2.3/suricata.html
https://docs.securityonion.net/en/2.3/sysmon.html
